### DISCUSSION TITLE
Any plans for supporting JSONpath?

### DISCUSSION THREAD

**Discussion by xR3b0rn** [nlohmann/json#4208] [Ideas]
https://goessner.net/articles/JsonPath/

**Comment by nlohmann** [comment]
Not by me, but PRs welcome.

**Reply by xR3b0rn** [reply]
JsonPath doesn't seem very standardized. Is the degree of standardization an acceptance criterion?

**Reply by nlohmann** [reply]
That was my reaction to it years ago. I wonder whether people are actually using it.

**Reply by xR3b0rn** [reply]
I mean, JsonPath is a superset of JsonPointer. So when peaople are using JsonPointer, they are also using JsonPath, aren't they?

**Reply by xR3b0rn** [reply]
Looks like an interesting task, however, if there is no demand (except me), I don't know if it is worth the effort.

**Reply by AdrianKarolewski** [reply]
Hello if this feature is still welcome, I would be happy to start coding. I think it can be a good point to start contributing.

**Reply by nlohmann** [reply]
No, it is still not planned. I guess it will be a huge PR. If anyone wants to work on this, I'd be happy to support.
